tests/unit/ReleaseNote.spec.js

```
@@ -0,0 +1,9 @@
+import { mount } from '@vue/test-utils'
+import ReleaseNote from '@/components/ReleaseNote.vue'
+
+describe('ReleaseNote.vue', () => {
```

ESLint

[error] 'describe' is not defined.

(no-undef)